source/api_documentation/network_integration/advertiser_campaigns/_get_advertiser_campaign_unarchive.rst

```diff
@@ -0,0 +1,15 @@
+.. container:: endpoint-long-description
```

Comment:
@mikeweaver 
Yeah it doesn't seem to follow convention if we're altering anything with GET. I think the problem is that we haven't been following convention for the history of the API.

@zmathe-sv 
Can you comment on this?